jbehave-support-core/src/main/java/org/jbehavesupport/core/web/WebSteps.java

```diff
@@ -350,4 +358,13 @@ private String getLastOpenedWindowHandler() {
         return handles.stream().skip(handles.size() - 1).findFirst().get();
     }
 
+    private void assertMandatoryColumns(ExamplesTable examplesTable, List<String> expectedColumns) {
+        SoftAssertions softly = new SoftAssertions();
+        expectedColumns.forEach(key -> {
+            if (examplesTable.getHeaders().stream().noneMatch(column -> column.equals(key))) {
+                softly.fail("Examples table must contain column'" + key + "'");
```

Comment:
Nitpick but missing space between column and key? :)

Reply:
Sorry didn't see this :)